zebrad/src/components/sync/end_of_support.rs

```diff
@@ -22,7 +22,8 @@ pub const ESTIMATED_RELEASE_HEIGHT: u32 = 2_471_000;
 ///
 /// - Zebra will exit with a panic if the current tip height is bigger than the `ESTIMATED_RELEASE_HEIGHT`
 ///  plus this number of days.
-pub const EOS_PANIC_AFTER: u32 = 112;
```

Comment:
We should revert and possibly decrease this value to ensure that there are no zebra versions running with old consensus rules after the end of the current protocol version which includes the devfund.